tensorboard/plugins/interactive_inference/witwidget/notebook/colab/wit.py

```diff
@@ -157,6 +156,12 @@ def infer_mutants(wit_id, details):
         window.spriteCallback(wit.localAtlasUrl);
       }}
     }};
+    // BroadcastChannel to allow examples to be updated by a call from an
+    // output cell that isn't the cell hosting the WIT widget.
+    const updateExampleListener = new BroadcastChannel('updateExamples');
```

Comment:
It seems that every instance of WIT is listening to the same broadcast channel. In case there are multiple WITs in the same colab, this would cause calling set_examples on one of them to update examples on all windows. A fix is to append wit id to channel name 'UpdateExamples_{}'.format(id).

Reply:
great catch, fixed